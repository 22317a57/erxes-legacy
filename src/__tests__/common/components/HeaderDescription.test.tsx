import { mount, shallow } from 'enzyme';
import React from 'react';

import HeaderDescription from '../../../modules/common/components/HeaderDescription';

describe('HeaderDescription component', () => {
  const defaultProps = {
    icon: 'email',
    title: 'Email',
    description: 'write your email'
  };

  test('renders HeaderDescription successfully', () => {
    shallow(<HeaderDescription {...defaultProps} />);
  });
});
