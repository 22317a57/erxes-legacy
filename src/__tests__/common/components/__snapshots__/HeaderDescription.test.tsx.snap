// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderDescription component snapshot matches 1`] = `
<div
  className="sc-iwsKbI kEjJKH"
>
  <img
    className="sc-gZMcBi eOUcUq"
    src="email"
  />
  <span>
    <h4>
      Email
    </h4>
    write your email
  </span>
</div>
`;
